src/controllers/localization.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const locale = require('locale');
+const localizationContent = require('../localizationContent');
+
+const _getSupportedLocales = () => {
+    const localesInContent = [];
+    for (const localeInContent in localizationContent.resources) {
+        localesInContent.push(localeInContent);
+    }
+
+    return new locale.Locales(localesInContent);
+};
+
+exports.getLocale = (req, res) => {
```

Comment:
this is done mostly to show that backend and frontend use the same localization file (so there will be no problem with storing error messages in the same place)